packages/http-server-js/package.json

```diff
@@ -58,14 +61,28 @@
     "yaml": "~2.7.0"
   },
   "devDependencies": {
+    "@types/cross-spawn": "~6.0.6",
     "@types/node": "~22.13.9",
     "@typespec/compiler": "workspace:^",
     "@typespec/http": "workspace:^",
+    "@typespec/http-specs": "workspace:^",
+    "@typespec/internal-build-utils": "workspace:^",
     "@typespec/openapi3": "workspace:^",
+    "@typespec/spector": "workspace:^",
     "@vitest/coverage-v8": "^3.0.7",
     "@vitest/ui": "^3.0.7",
+    "cross-spawn": "^7.0.6",
```

Comment:
this package doesn't seem used, maybe some other ones too

Reply:
Ah you're right - I was using it until I switched to internal-build tools `run`

Reply:
Removed execa as well.